aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+      users might expect an error message to disrupt what they are doing, since the error is important,
+      but want to wait with reading new chat messages until they are done typing their own message.
+    </p>
+
+    <section id="live_region_states_and_properties">
+      <h3>Live Region States and Properties</h3>
+
+      <p>
+        ARIA has the following attributes mark up live regions.
+      </p>
+
+      <table>
+        <thead>
+        <tr>
+        <th>State/property</th>
+        <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><code>aria-live</code></td>
+            <td>Enables a live region.</td>
+           </tr>
+          <tr>
+            <td><code>aria-atomic</code></td>
+            <td>Indicates what content to announce.</td>
+          </tr>
+          <tr>
+            <td><code>aria-relevant</code></td>
+            <td>Indicates which content changes are relevant.</td>
+          </tr>
+          <tr>
+            <td><code>aria-busy</code></td>
+            <td>Defers content updates. Not specific to live regions. See the <a href="#aria-busy">next section</a>.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <section id="enable_live_regions_with_aria-live">
+        <h4>Enable Live Regions with <code>aria-live</code></h4>
+
+        <p>
+          The <code>aria-live</code> attribute indicates that an element is a live region.
+          Some roles implicitly set <code>aria-live</code>; this is discussed in a later section.
+        </p>
+
+        <ul>
+          <li><code>assertive</code>: assistive technologies can interrupt the user to provide this information,
+          but will not move the current focus, so the user&#39;s work flow is not interrupted.</li>
+
+          <li><code>polite</code>: assistive technologies will provide this information after the user is done with their current task.</li>
+
+          <li><code>off</code>: assistive technologies will not provide this information unless this region has focus.</li>
+        </ul>
+
+        <p>
+          With the exception of a few roles, the default value of <code>aria-live</code> is <code>off</code>.
+        </p>
+
+        <p>
+          For example, when the user types into a search field, the page could update the page with search results as the user is typing.
+          To inform users of assistive technology that this has happened, a polite live region can be used.
+          When the user is done typing, and the search is complete, the user is informed of how many results were found,
+          without having to move focus away from the search field.
+        </p>
+
+        <pre><code>&lt;form role="search" aria-labelledby="search">
+ &lt;h2 id="search">Search&lt;/h2>
+ &lt;label>Search query: &lt;input type="search" name="q" oninput="updateSearch(event)">&lt;/label>
+ &lt;div id="search-result-status" role="region" aria-live="polite">&lt;/div>
```

Comment:
[`region` is a landmark role](https://w3c.github.io/aria/#region).

Need to discuss with group whether a live region can use `role="region"` without being a landmark.

If so, then the ARIA spec needs to allow this use specifically for live regions.

Or, if the live region in this example is truly intended to be a landmark, then in needs an accessible name (or else conformance tools are going to flag it as missing one).

If, however, the group decides that `role="region"` should not be allowed on live regions without turning them into landmarks, and if it is decided that the live region in this example should not be a landmark (really, I think the role here should be "status", but we haven't described that yet), then we need to know what role, if any, should go on this example's live region div. (No role, i.e. "generic"? Or "group", perhaps? Or should we just use "status" role before it is discussed?)

Reply:
if it's not named then it shouldn't have a `role=region`.  does this example even need a role?

Reply:
> does this example even need a role?

Nope. :) Decision on the call was to delete `role=region` and leave it with no role.
(The "Contacts list" example later on can keep region and label because it's ok as a true landmark).

I didn't want to state "No role" as the "best" choice without floating a bunch of other possibilities out there first - even the wacky possibility of changing the spec to allow live region elements to have a "non-landmark role=region". 😄